cmd/down.go

```diff
@@ -61,6 +62,22 @@ func Down() *cobra.Command {
 				return err
 			}
 
+			if all {
+				devs, err := utils.GetAllDevsFromManifest(manifest)
```

Comment:
why can't you iterate directly on `range manifest.Dev`?
Also, could you move this logic to its own function?